rscripts/YAL5-codon-pairs.R

```diff
@@ -0,0 +1,1455 @@
+                                                             gene_names,
+                                                             dataset,
+                                                             hd_file,
+                                                             gff_df,
+                                                             min_read_length,
+                                                             snapdisp,
+                                                             asite_disp_path,
+                                                             filter_for_frame,
+                                                             feature_of_interest,
+                                                             expand_width))
+
+    # Check for the presence of the feature of interest. expanded_feature
+    # being empty will cause problems with normalization
+
+    if (length(expanded_feature) == 0) {
+
+      print(paste("No occurrences of", feature_being_studied))
+
+      if (expand_width > 1) {
+
+        print("Use expand_width = 1L to check for occurrences near to start or stop codon")
+      }
+
+      return()
+    }
+
+    # Run ExpandedRegionNormalisation() to calculate the relative number of
+    # reads mapping to each position around the feature of interest
+
+    normalized_expand_list <- ExpandedRegionNormalisation(codon_pos_pair_i200,
+                                                          gene_names, dataset,
+                                                          hd_file, gff_df,
+                                                          min_read_length,
+                                                          snapdisp,
+                                                          asite_disp_path,
+                                                          filter_for_frame,
+                                                          feature_of_interest,
+                                                          expand_width)
+
+    # Run OverlayedTibble() to create an average of reads at positions at and
+    # around the feature of interest
+
+    overlayed_tibbles <- OverlayedTibble(codon_pos_pair_i200,
+                                         gene_names, dataset, hd_file,
+                                         gff_df, min_read_length,
+                                         snapdisp, asite_disp_path,
+                                         filter_for_frame,
+                                         feature_of_interest,
+                                         expand_width)
+
+    # Create a new tibble listing the feature being studeied, and the RelCount
+    # at position 0, ie RelCount at the feature of interest
+
+    feature_rel_use <- tibble(Feature = feature_being_studied,
+                              RelCount = filter(overlayed_tibbles,
+                                                overlayed_tibbles$Rel_Pos == 0)$RelCount)
```

Comment:
overlayed_tibbles$Rel_Pos -> overlayed_tibbles$RelPos

This will make the else statement work and produce a comparative table